packages/components/src/table/test/index.js

```diff
@@ -112,4 +112,46 @@ describe( 'TableCard', () => {
 			).toBeInTheDocument();
 		}
 	} );
+
+	it( 'should render headers having class is-left-aligned if isLeftAligned is set to true', () => {
+		mockHeaders[ 0 ].isLeftAligned = true;
+		mockHeaders[ 1 ].isLeftAligned = true;
+		mockHeaders[ 1 ].isNumeric = true;
+
+		render(
+			<TableCard
+				title="Revenue"
+				headers={ mockHeaders }
+				isLoading={ false }
+				rowsPerPage={ 5 }
+				totalRows={ 5 }
+			/>
+		);
+
+		expect( screen.getAllByRole( 'columnheader' )[ 0 ] ).toHaveClass(
+			'is-left-aligned'
+		);
+
+		expect( screen.getAllByRole( 'columnheader' )[ 1 ] ).toHaveClass(
+			'is-left-aligned'
+		);
+	} );
+
+	it( 'should render headers not having class is-left-aligned if isLeftAligned is not set and isNumeric is true', () => {
+		mockHeaders[ 2 ].isNumeric = true;
+
+		render(
+			<TableCard
+				title="Revenue"
+				headers={ mockHeaders }
+				isLoading={ false }
+				rowsPerPage={ 5 }
+				totalRows={ 5 }
+			/>
+		);
+
+		expect( screen.getAllByRole( 'columnheader' )[ 2 ] ).not.toHaveClass(
+			'is-left-aligned'
+		);
+	} );
```

Comment:
@htdat could we add one more test case, for when `isLeftAligned` is not set, and `isNumeric` is false (that it automatically sets the isLeftAligned class)?
As this would be the **real** change in functionality, given we previously supported `isLeftAligned`.

Reply:
> @htdat could we add one more test case, for when isLeftAligned is not set, and isNumeric is false (that it automatically sets the isLeftAligned class)?

😁 Makes sense. I would have added it in my previous commit. Done here a87a3ce